### DISCUSSION TITLE
Crash reporting

### DISCUSSION THREAD

**Discussion by Totengeist** [maforget/ComicRackCE#13] [General]
I noticed that crash reporting was removed in https://github.com/maforget/ComicRackCE/commit/bd2d31293b1755d6ea5f53aa8b1badfc48ed4b56. This is probably a good idea early on, but it might be helpful when the Community Edition starts having versioned releases. Setting up a catch page for this should be relatively simple, and I'd be happy to assist with that.

I know there's a reproducible crash with clipboard content. Can someone let me know the reproduction steps on that? I'm curious about what the crash reporter's web requests look like.

**Comment by maforget** [comment]
For you question about the crash with clipboard content, just see issue #1. Problem is the crash report wouldn't work in that situation. It will bypass the error catcher. This is a Windows crash.

You can simulate a crash by typing CRASH on the splash screen.
